docs/plugin-transform-runtime.md

```diff
@@ -154,6 +155,12 @@ export default function(instance, Constructor) {
 }
 ```
 
+### `absoluteRuntime`
+
+`boolean`, defaults to `false`.
+
+When enabled, the loader will get the path of modules being passed into this module using the `cwd` set by the `babel-loader`. For example, this allows you to run this preset via a CLI tool and specify the `cwd` as the root directory of the CLI tool instead of the `process.cwd`. This is useful when your tool may have references to things like `@babel/runtime` and the `process.cwd` doesn't.
```

Comment:
I feel like @loganfsmyth did a fantastic job at describing the behavior of this option in https://github.com/babel/babel/pull/8435, wonder if we can just adapt it here?